### DISCUSSION TITLE
Can't log-in in Console but I can I WebUI - qBittorrent

### DISCUSSION THREAD

**Discussion by EdorasRP-system** [community-scripts/ProxmoxVE#140] [Q&A] [answered]
I'm using the same password and user for the WebUI, I log-in with no problem but In the console it says its incorrect, why?

**Comment by Mellowlynx** [comment, marked as answer]
By default, you should be able to just open and it will auto login.
If you used the Advanced option, you were asked to set a root password, if you did enter one, use that one to login.

**Reply by EdorasRP-system** [reply]
> By default, you should be able to just open and it will auto login. If you used the Advanced option, you were asked to set a root password, if you did enter one, use that one to login.

That was it, I ended up changing root password for that lxc, thanks!
